packages/sra-report/src/contract_addresses/mainnet_addresses.ts

```diff
@@ -0,0 +1,5 @@
+export const addresses = {
+    WETH: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
```

Comment:
Do you think it makes sense to pull these from the ABI? Hardcoded is probably ok, just a bit magic.

Reply:
i think its probably ok

Reply:
Agreed with @BMillman19. Won't change anytime soon.